src/controllers/users.controller.js

```diff
@@ -173,25 +177,53 @@ const solves = async (request, response) => {
 };
 
 const ranking = async (request, response) => {
-    const dbRes = await pool.query(
```

Comment:
I mean, you could just disable the submissions after some timestamp instead of filtering the ranking.
Also, I think that the `freeze == 'true'` check does nothing, the entire if statement is redundant and not required.
You could just remove it and always filter by the timestamp. The result would be functionally the same.

Reply:
if statement is redundant, because freeze ha wrong value by mistake